bp1/bench.cc

```diff
@@ -298,73 +305,73 @@ void test(const unsigned int s,
   LIKWID_MARKER_START("matvec");
 #endif
   double matvec_time = 1e10;
-  for (unsigned int t=0; t<4; ++t)
+  for (unsigned int t = 0; t < 4; ++t)
     {
       time.restart();
-      for (unsigned int i=0; i<50; ++i)
+      for (unsigned int i = 0; i < 50; ++i)
         mass_operator.vmult(tmp, output);
-      data = Utilities::MPI::min_max_avg(time.wall_time(), MPI_COMM_WORLD);
-      matvec_time = std::min(data.max/50, matvec_time);
+      data        = Utilities::MPI::min_max_avg(time.wall_time(), MPI_COMM_WORLD);
+      matvec_time = std::min(data.max / 50, matvec_time);
     }
 #ifdef LIKWID_PERFMON
   LIKWID_MARKER_STOP("matvec");
 #endif
 
   if (Utilities::MPI::this_mpi_process(MPI_COMM_WORLD) == 0)
-    std::cout << std::setw(2) << fe_degree << " | " << std::setw(2) << n_q_points
-              << " | " << std::setw(10) << tria->n_global_active_cells()
-              << " | " << std::setw(11) << dof_handler.n_dofs()
-              << " | " << std::setw(11) << solver_time/solver_control.last_step()
-              << " | " << std::setw(11) << dof_handler.n_dofs()/solver_time2*solver_control.last_step()
-              << " | " << std::setw(11) << solver_time2/solver_control.last_step()
-              << " | " << std::setw(11) << solver_time3/solver_control.last_step()
-              << " | " << std::setw(11) << solver_time4/solver_control.last_step()
-              << " |"  << std::setw(3) << solver_its2
-              << " "   << std::setw(3) << solver_control.last_step()
-              << " | " << std::setw(11) << matvec_time
-              << std::endl;
+    std::cout << std::setw(2) << fe_degree << " | " << std::setw(2) << n_q_points << " | "
```

Comment:
This is likely a candidate for disabling clang-format. Maybe just put `//` at the end of these lines is nicer than a commit `//clang-format off`?

Reply:
Done! Any other problematic regions?